.eslintrc.json

```diff
@@ -53,7 +53,10 @@
         "prefer-const": "error",
         "quotes": ["error", "single", {"avoidEscape": true}],
         "@typescript-eslint/semi": ["error", "never"],
-        "@typescript-eslint/member-delimiter-style": {"delimiter": "semi", "requireLast": false},
```

Comment:
Do not change  `.eslintrc.json` at the same time as other changes to keep the history clean.

Reply:
It's not a change, it's a fix. The previous one is buggy for eslint v6. Obviously, the error level was missing.

Reply:
You must open a separate PR. This change causes the following errors:

```
$ ./node_modules/.bin/eslint --ext .ts .

/Users/tamura/src/github/LaTeX-Workshop/dev/createSymbolSvgs.ts
   5:18  error  Expected a comma          @typescript-eslint/member-delimiter-style
   6:23  error  Expected a comma          @typescript-eslint/member-delimiter-style
   7:20  error  Expected a comma          @typescript-eslint/member-delimiter-style
   8:21  error  Expected a comma          @typescript-eslint/member-delimiter-style
   9:23  error  Expected a comma          @typescript-eslint/member-delimiter-style
  10:18  error  Expected a comma          @typescript-eslint/member-delimiter-style
  11:22  error  Unexpected separator (;)  @typescript-eslint/member-delimiter-style
  43:47  error  Unexpected separator (;)  @typescript-eslint/member-delimiter-style
  44:11  error  Unexpected separator (;)  @typescript-eslint/member-delimiter-style
  62:52  error  Expected a comma          @typescript-eslint/member-delimiter-style
  63:55  error  Expected a comma          @typescript-eslint/member-delimiter-style
  64:51  error  Expected a comma          @typescript-eslint/member-delimiter-style
  65:60  error  Expected a comma          @typescript-eslint/member-delimiter-style
  66:51  error  Unexpected separator (;)  @typescript-eslint/member-delimiter-style

✖ 14 problems (14 errors, 0 warnings)
  14 errors and 0 warnings potentially fixable with the `--fix` option.

```